packages/server/src/fhir/routes.ts

```diff
@@ -339,7 +339,7 @@ protectedRoutes.use(
 
     const request: FhirRequest = {
       method: req.method as HttpMethod,
-      url: req.originalUrl.replace('/fhir/R4', ''),
+      url: stripPrefix(req.originalUrl, '/fhir/R4'),
```

Comment:
Do you remember what input required this change?

Reply:
In my local testing environment with ngrok, I had the API running on a sub-path, so `req.originalUrl` was e.g. `/api/fhir/R4/$graphql`